cni/deployments/kubernetes/install/test/install_cni.go

```diff
@@ -247,21 +296,36 @@ func doTest(testNum int, wd, preConfFile, resultFileName, expectedOutputFile,
 		docker("logs", containerID, t)
 		docker("rm", containerID, t)
 	}()
-	time.Sleep(10 * time.Second)
 
-	compareConfResult(testWorkRootDir, tempCNIConfDir, resultFileName, expectedOutputFile, t)
+	resultFile := tempCNIConfDir + "/" + resultFileName
+	if delayedConfFile != "" {
+		timeout := time.After(5 * time.Second)
```

Comment:
Same as for the ticker below, it'd be cleaner to use a `time.NewTimer()`. Not as a big deal as the ticker since this one will get garbage-collected eventually.